app/javascript/mastodon/features/onboarding/index.jsx

```diff
@@ -85,6 +86,10 @@ class Onboarding extends ImmutablePureComponent {
     this.setState({ step: null });
   };
 
+  handleNextClick = () => {
```

Comment:
Maybe use a more explicit name, like `handleProfileNextClick`, as this is a handled on a nested screen?